### PR TITLE
Initial Kymppi react-native pull into master

### PR DESCRIPTION
**Pull request by jahonen**

### CODE REVIEW

**Comment by villebro** [comment]
You need to (at least) remove `.expo/**`

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.expo
+node_modules
```
Comment (villebro) [inline .gitignore:2]: Add a newline

Comment (jahonen) [reply .gitignore:2]: done

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+.DS_Store
+assets/.DS_Store
+src/.DS_Store
```
Comment (villebro) [inline .gitignore:5]: no need to specify these separately, just `.DS_Store` will do.

**Pull request merged**